idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/settings/tabs/export/ImageVectorExportSettingsScreen.kt

```diff
@@ -142,5 +148,6 @@ private fun ImageVectorExportSettingsPreview() = PreviewTheme(alignment = Alignm
         useFlatPackage = true,
         useExplicitMode = false,
         addTrailingComma = false,
+        fileIndent = 4,
```

Comment:
Hardcode?

Reply:
Its just for @Composable preview, i think we don't need use setting's indent size

Reply:
Makes sense.